stl/inc/iterator

```diff
@@ -186,6 +186,44 @@ _NODISCARD insert_iterator<_Container> inserter(_Container& _Cont, typename _Con
     return insert_iterator<_Container>(_Cont, _Where);
 }
 
+#ifdef __cpp_lib_concepts
+// CLASS TEMPLATE move_sentinel
+template <semiregular _Se>
+class move_sentinel {
+public:
+    constexpr move_sentinel() = default;
+
+    constexpr explicit move_sentinel(_Se _Val) noexcept(is_nothrow_move_constructible_v<_Se>) // strengthened
+        : _Last(_STD move(_Val)) {}
+
+    // clang-format off
+    template <class _Se2>
+        requires convertible_to<const _Se2&, _Se>
+    constexpr move_sentinel(const move_sentinel<_Se2>& _Val)
+        noexcept(is_nothrow_constructible_v<_Se, const _Se2&>) // strengthened
+        : _Last{_Val._Get_last()} {}

```

Comment:
I observe that you're using braces here, but parens on line 197. Should they be consistent?